.github/workflows/esp32-mkimage.yaml

```diff
@@ -31,13 +31,33 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+
+  # This job is used to determine the esp-idf version to build with.
+  # For maintenance branch release-0.6 we want to continue to use esp-idf v5.3.x to build.
+  # New releases should be build with esp-idf v5.4.x to enable the latest features for new
+  # chips; for example, esp-wifi-external to allow ESP32-P4 chips to use esp-wifi-hosted
+  # firmware on an on-board ESP32C6 for WiFi.
+  use-esp-idf:
+    name: Find IDF version
+    runs-on: ubuntu-22.04
+    outputs:
+      idf_version: ${{ steps.find-version.outputs.IDF-VERSION }}
+    steps:
+      - id: find-version
+        run: |
+          if [ ${{ github.ref == 'refs/heads/release-0.6' || github.event.release.target_commitish == 'release-0.6'}} ]; then
```

Comment:
Sorry for this last minute comment:
This conditional is not required, since we are going to merge this PR into main, and we never merge back main into release-0.6 (we always do instead `release-0.6` -> `main`).
So we either remove this conditional since `${{ github.ref == 'refs/heads/release-0.6' || github.event.release.target_commitish == 'release-0.6'}}` will always be false in main, or rebase this into release-0.6.
I think we can go with the first option by the way.

Reply:
This is fixed now.